utils/ips/claimed_ip_port.go

```diff
@@ -50,13 +50,13 @@ func NewClaimedIPPort(
 		AddrPort:  ipPort,
 		Timestamp: timestamp,
 		Signature: signature,
-		NodeID:    ids.NodeIDFromCert(cert),
+		NodeID:    ids.ShortNodeIDFromCert(cert).NodeID(),
 	}
 
 	packer := wrappers.Packer{
 		Bytes: make([]byte, preimageLen),
 	}
-	packer.PackFixedBytes(ip.NodeID[:])
+	packer.PackFixedBytes(ip.NodeID.Bytes())
 	packer.PackLong(timestamp)
 	ip.GossipID = hashing.ComputeHash256Array(packer.Bytes)
```

Comment:
This looks dubious. If `ip.NodeID.Bytes()` started returning `32` bytes, it seems like this may behave unexpectedly.

I need to look into this a bit more carefully.

Reply:
in the context of this PR, I believe that this change is safe. The `ip.NodeID` is populated few lines above as `NodeID:    ids.ShortNodeIDFromCert(cert).NodeID()`. Hence, guarantee that it's a 20 bytes slice.
Future PRs would need to update this logic, naturally.